bookkeeper-server/src/main/proto/BookkeeperProtocol.proto

```diff
@@ -83,6 +92,7 @@ message Request {
     optional ReadLacRequest readLacRequest = 104;
     optional GetBookieInfoRequest getBookieInfoRequest = 105;
     optional StartTLSRequest startTLSRequest = 106;
+    optional SyncRequest syncRequest = 107;
```

Comment:
FSyncJournalRequest

Reply:
This won't be used only for journalled bookies

 @jvrao

Reply:
Anche for this @sijie?

Reply:
And for this?

Reply:
SyncLedgerRequest or LSyncRequest? be more specific it is a per ledger operation?

Reply:
ForceRequest? force seems clearer to me. sync is suggesting something more akin to flushing. @sijie @eolivelli

Reply:
force sounds fine to me. it is a java convention

Reply:
OK, I will rename Sync with Force, so ForceRequest, FORCE, WriteHandle#force()....

I will wait for some time before actually performing the rename, I am preparing other patches and this will be very disruptive

 @jvrao Are you OK with this new name 'force' instead of 'sync' ?